### PR TITLE
Update Arch packaging README.md with link and package mgr options

### PR DESCRIPTION
**Pull request by skymoore**
##### SUMMARY

The suggested package manager is unsecure, I felt it my duty to mention a better option.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

packaging
##### ANSIBLE VERSION

```
N/A

```

##### ADDITIONAL INFORMATION

See the links in the change, and this pull requests summary.

```
N/A
```

### CODE REVIEW

**Comment by gundalow** [comment]
@bcoca I believe you are an Arch user

**Comment by maxamillion** [comment]
I think `X is a much better package manager` for basically any value of X is going to be subjective even if you can mathematically prove that the depsolving is better or runtime is faster, there's going to be users with preferences and preconceived biases. I prefer not to make such claims in documentation without some quantifier(s) about what `much better` pertains to.

**Comment by bcoca** [comment]
@gundalow do not pull me into an Arch package manager ... i survived the rpm/deb/tgz package conflict, the apt/yum split and BSDs package manager flame pit ... I don't have the strength for another

**Comment by maxamillion** [comment]
@gundalow @bcoca feel free to throw me under the distro packager bus if/when necessary

**Comment by bcoca** [comment]
@maxamillion i did that already when I passed my work on ansible-galaxy to you.

**Comment by acozine** [comment]
@mskymoore Could you update your PR to skip the comparative assessment? I agree with @maxamillion that Ansible docs shouldn't make claims about which package managers are better or worse, but I'd be happy to merge something like "You can also use `pikaur` to install `ansible-git`" with the example command.

**Comment by skymoore** [comment]
Sorry friends, fell off the face of the earth for a while, but will be sure and respond promptly next time.  I agree that my comment was subjective.  Thanks to acozine for the edit!

File: packaging/arch/README.md [hunk 1]
```diff
@@ -2,12 +2,19 @@ Arch Packaging Files
...
 
-  [1]: https://aur.archlinux.org/packages/ansible-git/
-  [2]: https://aur.archlinux.org/packages/ansible/
+    pikaur -S ansible-git
 
+You can also install the stable version of [ansible][3] as an AUR package.
+
+  [1]: https://aur.archlinux.org/packages/ansible-git/
+  [2]: https://wiki.archlinux.org/index.php/AUR_helpers
+  [3]: https://aur.archlinux.org/packages/ansible/
```
Comment (samccann) [inline packaging/arch/README.md]: This link doesn't seem to work?

**Pull request merged**